### PR TITLE
Add section for Ledger support

### PR DESCRIPTION
**Pull request by david-yan**
Resolves https://github.com/oasislabs/docs/issues/185

### CODE REVIEW

File: src/hsm/ledger.md [hunk 1]
```diff
@@ -0,0 +1,69 @@
+# Ledger Support
+
+Ledger is one of the HSM platforms that Oasis supports for phase 1 of the
```
Comment (ravenac95) [inline src/hsm/ledger.md]: @Afauvre what's the canonically public way to refer to phase 1?

**Comment by Afauvre** [comment]
Let's just call it "Mainnet Launch"

File: src/hsm/ledger.md [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+network.
```
Comment (ravenac95) [inline src/hsm/ledger.md]:
```suggestion
```

File: src/hsm/ledger.md (hunk 1 above)
Comment (ravenac95) [inline src/hsm/ledger.md]:
```suggestion
Ledger is one of the HSM platforms that Oasis supports for Mainnet Launch.
```

File: src/hsm/ledger.md [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+Before following this guide, make sure you've followed the [Prerequisites
+Guide](./prerequisites.md) to install the `oasis-node` binary.
+
+## Installation
+
+To use with the Ledger App, install the Ledger application from
+the Ledger Live Manager. You will have to set developer mode to 'on' in order
+to find it.
+
+::: warn NOTE
```
Comment (ravenac95) [inline src/hsm/ledger.md]:
```suggestion
::: warning NOTE
```

I believe that's how to get it to render properly?

File: src/hsm/ledger.md [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+  --signer ledger \
+  --signer.ledger.address oasis19hpt4y2reqwyfqcd53asjchdqf468chr673y6jn07xjp36w32jlscf0me \
+  --signer.ledger.index 1 \
+...
+```
+
+::: note Note
+The `--signer.dir` flag is still necessary for the command to run, but it is
+not used. The specified directory must contain a valid `entity.json` file.
+
```
Comment (ravenac95) [inline src/hsm/ledger.md]:
```suggestion
:::

```

**Comment by ravenac95** [comment]
@david-yan what's the best way for us to test these docs?

**Comment by david-yan** [comment]
@ravenac95 by test do you mean dogfood? I only have one device, but I can give it to someone to try to sign transactions using these docs.

File: src/hsm/ledger.md [hunk 5]
```diff
@@ -0,0 +1,69 @@
...
+  --signer.dir entity \
+  --signer ledger \
+  --signer.ledger.address oasis19hpt4y2reqwyfqcd53asjchdqf468chr673y6jn07xjp36w32jlscf0me \
+  --signer.ledger.index 1 \
+...
+```
+
+::: note Note
+The `--signer.dir` flag is still necessary for the command to run, but it is
+not used. The specified directory must contain a valid `entity.json` file.
```
Comment (ravenac95) [inline src/hsm/ledger.md]: Make a note for now this needs to be a valid entity.json file but the values in the file are ignored. (we should just provide a dummy file here)
```suggestion
not used. The specified directory must contain a valid `entity.json` file. For now,
use this dummy entity.json:

{"id":"AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA=","nodes":null,"allow_entity_signed_nodes":false}

```

File: src/hsm/ledger.md [hunk 6]
```diff
@@ -0,0 +1,69 @@
...
+`--signer.ledger.address <oasis_app_address>` flags. The
+`--signer.ledger.index` flag can be specified for an alternate account index.
+For example:
+
+```bash
+oasis-node stake account gen_burn \
+  --signer.dir entity \
+  --signer ledger \
+  --signer.ledger.address oasis19hpt4y2reqwyfqcd53asjchdqf468chr673y6jn07xjp36w32jlscf0me \
+  --signer.ledger.index 1 \
```
Comment (ravenac95) [inline src/hsm/ledger.md]:
```suggestion
  --signer.ledger.index 1 \
  --genesis.file $GENESIS_PATH \
  --transaction.fee.amount 1 \
  --transaction.fee.gas 1000 \
  --transaction.file burn.tx
```

File: src/hsm/ledger.md [hunk 7]
```diff
@@ -0,0 +1,69 @@
...
+
+To sign transactions with the Ledger App, run the `oasis-node stake account
+gen_<transaction_type>` command, and specify the `--signer ledger` and
+`--signer.ledger.address <oasis_app_address>` flags. The
+`--signer.ledger.index` flag can be specified for an alternate account index.
+For example:
+
+```bash
+oasis-node stake account gen_burn \
+  --signer.dir entity \
```
Comment (ravenac95) [inline src/hsm/ledger.md]: We should make this obvious that this is a path to a directory containing an entity.json

File: src/hsm/ledger.md [hunk 8]
```diff
@@ -0,0 +1,69 @@
...
+  --signer ledger \
+  --signer.ledger.address oasis19hpt4y2reqwyfqcd53asjchdqf468chr673y6jn07xjp36w32jlscf0me \
+  --signer.ledger.index 1 \
+...
```
Comment (ravenac95) [inline src/hsm/ledger.md]:
```suggestion
```

File: src/hsm/ledger.md (hunk 7 above)
Comment (ravenac95) [reply src/hsm/ledger.md]: The note below is useful but perhaps just do something like 
```suggestion
  --signer.dir /path/to/entity \
```

**Comment by ravenac95** [comment]
@david-yan tested this and it works. We just need to address some of the other comments.

File: src/hsm/ledger.md [hunk 9]
```diff
@@ -0,0 +1,69 @@
...
+```
+
```
Comment (david-yan) [inline src/hsm/ledger.md]:
```suggestion
[//]: <> (TODO: Replace this section with one detailing how to extract the entity from the Ledger device, after the tooling is added.
```

**Pull request merged**